### DISCUSSION TITLE
ERR [File Watcher (node.js)] Error: EACCES: permission denied, stat '/home/codeserver/MyDir.code-workspace

### DISCUSSION THREAD

**Discussion by Shahi-Paneer** [coder/code-server#4373] [Q&A]
Unable to do anything with code-server as it won't let me write to anywhere. 

`Unable to write file 'vscode-remote://liemcomputing.ca/home/codeserver/MyDir.code-workspace' (NoPermissions (FileSystemError): Error: EACCES: permission denied, open '/home/codeserver/MyDir.code-workspace')`

Any help would be greatly appreciated. 

I just need to give it write access to a certain folder within /home.

Edit:

Running on:  

NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

**Comment by code-asher** [comment]
How are you spawning code-server?  The user you use to spawn code-server will need permissions to that directory.
